x-pack/platform/plugins/shared/streams_app/test/scout/ui/tests/data_management/data_quality.spec.ts

```diff
@@ -73,8 +124,89 @@ test.describe('Stream data quality', { tag: ['@ess', '@svlOblt'] }, () => {
     await pageObjects.streams.verifyDatePickerTimeRange(dataQualityTimeRange);
   });
 
-  test('should edit failure store for wired stream from data quality tab', async ({ page }) => {
-    // Click on the Failed documents card to expand it
+  test('should toggle between degraded and failed docs quality issues charts', async ({ page }) => {
+    // Default chart should be for degraded docs
+    await expect(page.getByTestId('datasetQualityDetailsLinkToDiscover')).toBeVisible();
+
+    // Click to show failed docs chart
+    await page.getByTestId('datasetQualityDetailsSummaryKpiCard-Failed documents').click();
+  });
+
+  test('should show degraded fields table with data', async ({ page }) => {
+    // Quality issues table should be visible
+    const degradedFieldTable = page.getByTestId('datasetQualityDetailsDegradedFieldTable');
+    await expect(degradedFieldTable).toBeVisible();
+
+    // Should show table headers (scope to the table to avoid ambiguity)
+    await expect(degradedFieldTable.getByRole('columnheader', { name: 'Field' })).toBeVisible();
+    await expect(degradedFieldTable.getByRole('columnheader', { name: 'Documents' })).toBeVisible();
+    await expect(degradedFieldTable.getByRole('columnheader', { name: 'Issue' })).toBeVisible();
+    await expect(
+      degradedFieldTable.getByRole('columnheader', { name: 'Last occurred' })
+    ).toBeVisible();
+
+    // Verify there is at least one degraded field row
+    const degradedFieldRows = degradedFieldTable.locator('tbody tr');
+    expect(await degradedFieldRows.count()).toBeGreaterThan(0);
```

Comment:
do we need to await on this expect? the test might just ignore it and pass right?

await expect(degradedFieldRows).toHaveCount({ min: 1 });